src/utils/digest.ts

```diff
@@ -0,0 +1,88 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import { createHash } from "crypto";
+
+function decimalToHex(d: number, padding: number) {
+    let hex = Number(d).toString(16);
+    padding = typeof (padding) === "undefined" || padding === null ? padding = 2 : padding;
+
+    while (hex.length < padding) {
+        hex = "0" + hex;
+    }
+
+    return hex;
+}
+
+export const parseHeaderWWWAuthenticateForDigest = (v: string): { [s: string]: any } => {
```

Comment:
`[s: string]: any` can be `[s: string]: string`, right?

Reply:
yes it is